tox.ini

```diff
@@ -240,6 +240,14 @@ commands =
         --cov-config=certbot-ci/certbot_integration_tests/.coveragerc
     coverage report --include 'certbot/*' --show-missing --fail-under=62
 
+[testenv:integration-dns-rfc2136]
+commands =
+    {[base]pip_install} acme certbot certbot-dns-rfc2136 certbot-ci
+    pytest certbot-ci/certbot_integration_tests/rfc2136_tests \
+        --acme-server={env:ACME_SERVER:pebble} \
```

Comment:
At that point, since the tests will not work with Pebble or Challtestsrv, I think it would be clearer to hardcode the flag values.